openslides_backend/models/models.py

```diff
@@ -1204,6 +1204,15 @@ class Poll(Model):
     )
     meeting_id = fields.RelationField(to={Collection("meeting"): "poll_ids"})
 
+    CREATED = "created"
+    STARTED = "started"
+    FINISHED = "finished"
+    PUBLISHED = "published"
+
+    ANALOG_TYPE = "analog"
+    NAMED_TYPE = "named"
+    PSEUDOANONYMOUS_TYPE = "pseudoanonymous"
```

Comment:
You cannot just add code to the `models.py`, it will be overwritten on the next model generation. Please take a look at https://github.com/OpenSlides/openslides-backend/blob/06f629b99d979f807584598ab5f7d310a767995b/cli/generate_models.py#L169 to see how you can insert these enums in this file.

Also, I would prefer the name to clearly state what is enum'd here, like `STATE_CREATED` and `TYPE_ANALOG`.

Reply:
Added the lines to the `ADDITIONAL_MODEL_CODE`.
Renamed the enum entries.